src/scripts/filtering.json

```diff
@@ -11,6 +11,12 @@
         "name": "hide your own posts",
         "tooltip": "hide your own posts on the dashboard"
       },
+      "hideShares": {
+        "value": true,
+        "type": "toggle",
+        "name": "hide shares toggle",
+        "tooltip": "adds a toggle on home feed to hide shares of accounts you don't follow. by @wowperfect"
```

Comment:
if it specifically hides shares of accounts you don't follow, wouldn't it need to check the following state to do so?

Reply:
maybe better phrasing would be "hide shares except of someone you follow sharing their own post" except that sounds a bit wordy to me. 

thoughts?

Reply:
just "hide shares" might be the best way to describe it. you will still get some opaque and transparent shares, but they're either contentful or transparent shares of contentful posts still all from the same blog. 

in that regard, i forgot to add it during the review, but it currently seems to only check the current share with the last index in the `shareTree`, which means that resharing your share of someone else's post would cause it to still show up with shares hidden. i would do something like:
```js
if (model.shareTree.some(post => post.postingProject.handle !== poster)) $(post).parent().parent().addClass(isShareClass);
```

Reply:
oo good idea thx